lib/buffer-source.js

```diff
@@ -75,6 +75,28 @@ function createBufferSource (src, opt) {
         return duration
       }
     },
+    currentTime: {
+      enumerable: true, configurable: true,
+      get: function () {
+        if (!playing) 
+          return audioCurrentTime
+        else
+          return audioCurrentTime + (rightNow() - audioStartTime) / 1000
```

Comment:
I guess buffer source time is better be based on [audioContext.currentTime](https://developer.mozilla.org/en-US/docs/Web/API/AudioContext/currentTime) than "browser time"